extension/intents/bookmarks/bookmarks.js

```diff
@@ -72,3 +72,15 @@ intentRunner.registerIntent({
     }
   },
 });
+
+intentRunner.registerIntent({
+  name: "bookmarks.remove",
+  async run(context) {
+    const activeTab = await context.activeTab();
+    const metadata = await pageMetadata.getMetadata(activeTab.id);
+    const bookmarks = await browser.bookmarks.search({ url: metadata.url });
```

Comment:
This part is a little subtle: pageMetadata gets the [canonical URL](https://yoast.com/rel-canonical/), while `activeTab.url` will be the URL in the URL bar. For many pages this will be the same, but it would be better to use `activeTab.url` here since it's the most likely thing that was bookmarked.